jfrog-cli/docs/artifactory/gorecursivepublish/help.go

```diff
@@ -0,0 +1,8 @@
+package gorecursivepublish
+
+const Description = "Populates and publish dependencies and transitive dependencies of the project"
```

Comment:
I suggest to rephrase the description to:
"Recursively discovers all project dependencies, both direct and indirect, and publishes them to Artifactory."

Reply:
Done